libraries/Microsoft.Bot.Builder.AI.LUIS/LuisRecognizerOptions.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Bot.Builder.AI.Luis
+{
+    public abstract class LuisRecognizerOptions
+    {
+        protected LuisRecognizerOptions(LuisApplication application)
+        {
+            Application = application ?? throw new ArgumentNullException(nameof(application));
+        }
+
+        public LuisApplication Application { get; private set; }
+
+        /// <summary>
+        /// Gets or sets the time in milliseconds to wait before the request times out.
+        /// </summary>
+        /// <value>
+        /// The time in milliseconds to wait before the request times out. Default is 100000 milliseconds.
+        /// </value>
+        public double Timeout { get; set; } = 100000;
```

Comment:
Why? Can we use TimeSpan instead of milliseconds? TimeSpan.FromSeconds(3) is much nicer :)

Reply:
And why did we choose the 100 second value? 3 minutes seems like a lot to wait for a bot to reply

Reply:
This is the default in the current LuisRecognizer. I kept the same to reduce friction and any possible change on behavior.